docs/http.rst

```diff
@@ -92,6 +92,9 @@ Unless otherwise specified, all calls return the following error codes:
    - errno 110 - Invalid crypto keys specified
    - errno 111 - Missing Required Header
 
+       - Missing TTL Header - Include the Time To Live header (`ref<https://tools.ietf.org/html/draft-ietf-webpush-protocol#section-6.2>`_)
+       - Missing Crypto Headers - Include the appropriate encryption headers (`ref<https://webpush-wg.github.io/webpush-encryption/#rfc.section.3.2>`_ and `ref<https://tools.ietf.org/id/draft-thomson-webpush-vapid-02.html#rfc.section.4>`_)
```

Comment:
Did you render this locally to see that it works? I'm asking because this doesn't look like valid rst. 'ref' is a special Sphinx markdown for doing a internal project link, and needs to look a bit different. An external link is close to this though, and will result in a link saying 'ref' that is hyper-linked. Did you want all these links to show up as 'ref'?